docs/src/main/sphinx/connector/vertica.md

```diff
@@ -218,17 +218,3 @@ The connector supports pushdown for a number of operations:
 
 ```{include} join-pushdown-enabled-false.fragment
 ```
-
-### Table statistics
```

Comment:
This connector supports reading statistics. We don't need to remove the section.

Reply:
Updated